src/thumbnailer.cc

```diff
@@ -171,16 +181,17 @@ Thumbnailer::Status Thumbnailer::GenerateAnimationEqualPSNR(
     enc_ = WebPAnimEncoderNew(frames_[0].pic.width, frames_[0].pic.height,
                               &anim_config_);
     if (enc_ == nullptr) return kMemoryError;
-
     // For each frame, find the quality value that produces WebPPicture
     // having PSNR close to target_psnr.
     for (auto& frame : frames_) {
       int frame_min_quality = 0;
       int frame_max_quality = 100;
       int frame_final_quality = -1;
 
-      const int frame_lowest_psnr = GetPSNR(&frame.pic, frame_min_quality);
-      const int frame_highest_psnr = GetPSNR(&frame.pic, frame_max_quality);
+      frame.config.quality = 0;
+      const int frame_lowest_psnr = GetPSNR(&frame.pic, frame.config);
```

Comment:
Since `FrameData` contains a `WebPConfig`, `GetPSNR()` could have a single `FrameData` argument now. Up to you.